.gitignore

```diff
@@ -0,0 +1,2 @@
+*.json
```

Comment:
We should probably also add a `.clang-format`

Reply:
Maybe. For the moment, I just use the default behavior of `clang-format` and I think that's ok.

Reply:
On the other hand, it would cost much to add the usual one.  But either way is fine.